modules/openapi-generator/src/main/resources/Java/libraries/feign/ApiClient.mustache

```diff
@@ -187,6 +187,7 @@ public class ApiClient {
     {{/joda}}
     objectMapper.registerModule(new JavaTimeModule());
     {{#openApiNullable}}
+    objectMapper.registerModule(new RFC3339JavaTimeModule());
```

Comment:
I guess this new line shouldn't be inside the `{{#openApiNullable}}` block, right?

Reply:
I see you marked this comment as resolved. But the code isn't changed. Maybe you have the changes locally and still need to push them?

Reply:
Yes, just pushed now